### DISCUSSION TITLE
How do I clean up the "old" 2022 version of DiscordChatExporter that used dotnet on Debian?

### DISCUSSION THREAD

**Discussion by TomLewis** [Tyrrrz/DiscordChatExporter#1259] [Q&A]
I cant remember how I had to install DiscordChatExporter but my cron includes dotnet, do I literally delete the DiscordChatExporter folder and apt remove dotnet? I cant remember its dependencies, im surprised its still working all this time later!

**Comment by Tyrrrz** [comment]
> do I literally delete the DiscordChatExporter folder and apt remove dotnet

That should work, assuming you used `apt` to install `dotnet`
